packages/react-components/src/index.ts

```diff
@@ -30,17 +30,19 @@ export {
   Overlay,
 } from './atoms';
 export {
-  OrcidSigninButton,
   GoogleSigninButton,
   Header,
   LabeledDateField,
+  LabeledDropdown,
+  LabeledPasswordField,
   LabeledTextArea,
   LabeledTextField,
-  LabeledPasswordField,
-  LabeledDropdown,
+  MenuButton,
+  OrcidSigninButton,
+  PeopleCard,
+  ProfilePersonalText,
```

Comment:
I believe we should not export this, it is only to be used as part of other components and does not make much sense standalone. Basically everything that we want to show standalone in storybook needs to be exported, nothing else. This is not technically enforced and possibly not consistent at the moment, but seems like the best guiding principle to me.